wamp/options.go

```diff
@@ -21,6 +21,7 @@ const (
 	OptPPTSerializer   = "ppt_serializer"
 	OptPPTCipher       = "ppt_cipher"
 	OptPPTKeyId        = "ppt_keyid"
+	OptSticky          = "sticky"
```

Comment:
unrelated?

Reply:
Related, but not yet supported :)

Reply:
That is tricky and requires more work, so I postpone it for now.

Situation: shared registration of PRC with not sticked invocation policy (e.g. Round robin) + progressive call + longtime running callees.

For single call it is easy — callee is chosen once upon invocation and message is routed to it. Thats all.

But progressive call consists of multiple messages. 
So there can be 2 situations: 
* all chunks of data are routed to the same callee (treating the whole call as one). This is a default behaviour.
* Every new chunk triggers a callee selection based on the invocation policy by specifying `sticky=false`. In this case router should keep info about all callees involved into the call + in case of call cancelling — router should cancel all callees respecting cancelling option and so on... Not so easy :) 

So the second flow is not implemented for now.

Reply:
Thanks for explaining.